File: Python/instancias.py
class Persona:

    def __init__(self, nombre, edad):
        self.nombre = nombre
        self.edad = edad
    
    def show_info(self):
        print(f"Hola me llamo {self.nombre} y tengo {self.edad}")


if __name__ == "__main__":
    kevin = Persona("Kevin", 17)
    kevin.show_info()
    

File: C++/instancias.cpp

#include <stdio.h>
#include <iostream>

using namespace std;



class Programador{
    private:
        string nombre;
        int edad;
        string skill;
    public:
        Programador(string _nombre, int _edad, string _skill){
            nombre = _nombre;
            edad = _edad;
            skill = _skill;
        }
    
    void getInfo(){
        cout << "Nombre " << nombre << " Edad " << edad << " Skill " << skill << endl;
    }
};

int main(int argc, char const *argv[])
{
    Programador kevin("Kevin", 17, "Programar");
    kevin.getInfo();
}
